packages/codemirror/src/theme.ts

```diff
@@ -156,7 +157,8 @@ export const jupyterHighlightStyle = HighlightStyle.define([
  */
 export const jupyterTheme: Extension = [
   jupyterEditorTheme,
-  syntaxHighlighting(jupyterHighlightStyle)
+  syntaxHighlighting(jupyterHighlightStyle),
+  ViewPlugin.fromClass(PythonBuiltin, { decorations: v => v.decorations })
```

Comment:
This is not the best place for loading the extension as ideally we should load it as a sidecar extension when using the Python language parser.

To achieve that, you can modify at 

https://github.com/jupyterlab/jupyterlab/blob/e40ed2ecd299e70c95dea1cd4ef233dfd42cee92/packages/codemirror/src/language.ts#L474-L477

```patch
-    return m.python(); 
+    return new LanguageSupport(m.pythonLanguage, [
+      ViewPlugin.fromClass(PythonBuiltin, { decorations: v => v.decorations })
+    ])
```

You will need to apply the same trick at https://github.com/jupyterlab/jupyterlab/blob/e40ed2ecd299e70c95dea1cd4ef233dfd42cee92/packages/codemirror/src/language.ts#L488